Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -164,13 +167,27 @@ public static string FindRichestPerson(string[] peopleAndBalances)
             for (int i = 0; i <= peopleAndBalances.Length - 1; i++)
             {
                 var currentPersonData = peopleAndBalances[i].Split(',');
-               
+
                 if (!decimal.TryParse(currentPersonData[currentPersonData.Length - 1], out decimal highestAmountOfPerson))
```

Comment:
Dirty. Split into two lines.

Reply:
oke, osmond told me that he thought you would do it this way.